src/components/feedback-message/FeedbackMessage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { WarningIcon, InfoIcon } from '../icon';
+import { Tooltip, TooltipTrigger } from '../tooltip';
+import styles from './FeedbackMessage.css';
+
+const IconRenderer = ({ type }) => {
+    if (type === 'error') {
+        return <WarningIcon className={ styles.icon } />;
+    }
+
+    return <InfoIcon className={ styles.icon } />;
+};
+
+IconRenderer.propTypes = {
+    type: PropTypes.oneOf(['error', 'info']),
+};
+
+const TooltipRenderer = ({ type, children }) => {
```

Comment:
Can we rename this to `FeedbackTooltip`?

Reply:
Thank you for those, I was having trouble coming up with a better name for these components 😅